src/main/java/service/DataConverterImpl.java

```diff
@@ -0,0 +1,22 @@
+package service;
+
+import core.basesyntax.FruitTransaction;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : data) {
+            String[] parts = line.split(",");
+            FruitTransaction.Operation operation
+                    = FruitTransaction.Operation.valueOf(parts.toString());
```

Comment:
It seems like you're trying to convert the first element of the `parts` array to an `Operation` enum, but `parts.toString()` will give you the string representation of the array object, not the contents of the array. You should use `parts[0]` to access the first element.